build.wake

```diff
@@ -13,100 +13,103 @@
 from wake import _
 
 # Useful build variants (arguments to all)
 def toVariant = match _
     "default" = Pass (Pair "native-cpp14-release" "native-c11-release")
-    "static"  = Pass (Pair "native-cpp14-static"  "native-c11-static")
-    "debug"   = Pass (Pair "native-cpp14-debug"   "native-c11-debug")
-    "wasm"    = Pass (Pair "wasm-cpp14-release"   "wasm-c11-release")
+    "static" = Pass (Pair "native-cpp14-static" "native-c11-static")
+    "debug" = Pass (Pair "native-cpp14-debug" "native-c11-debug")
+    "wasm" = Pass (Pair "wasm-cpp14-release" "wasm-c11-release")
     s = Fail "Unknown build target ({s})".makeError
 
+
 export def build: List String => Result String Error = match _
     "tarball", Nil = tarball Unit | rmap (\_ "TARBALL")
     kind, Nil =
         require Pass variant = toVariant kind
         all variant | rmap (\_ "BUILD")
     _ = Fail "no target specified (try: build default/debug/tarball)".makeError
 
+
 export def install: List String => Result String Error = match _
     dest, kind, Nil = doInstall (in cwd dest) kind | rmap (\_ "INSTALL")
     dest, Nil = doInstall (in cwd dest) "default" | rmap (\_ "INSTALL")
     _ = Fail "no directory specified (try: install /opt/local)".makeError
 
+
 # Build all wake targets
 def targets =
-    buildWake, buildWakeBox, buildFuseDaemon, buildShim, buildBSP, buildLSP, buildJobCache, buildWakeFormat, Nil
+    buildWake,
+    buildWakeBox,
+    buildFuseDaemon,
+    buildShim,
+    buildBSP,
+    buildLSP,
+    buildJobCache,
+    buildWakeFormat,
+    Nil
+
 
 def all variant =
-    require Pass x =
-        map (_ variant) targets
-        | findFail
+    require Pass x = map (_ variant) targets | findFail
     Pass (flatten x)
 
+
 # Install wake into a target location
 def doInstall dest kind =
     require Pass variant = toVariant kind
-    require Pass datfiles =
-        sources "{here}/share" `.*`
+    require Pass datfiles = sources "{@here}/share" `.*`
     def binfilesResult = all variant
     def releaseBin exe = installAs "{dest}/{replace `\.[^.]*$` '' exe.getPathName}" exe
-    require Pass datinstall =
-        findFailFn (installIn dest ".") datfiles
-    require Pass readmeSource =
-        source "README.md"
-    require Pass readme =
-        installIn "{dest}/share/doc/wake" "." readmeSource
+    require Pass datinstall = findFailFn (installIn dest ".") datfiles
+    require Pass readmeSource = source "README.md"
+    require Pass readme = installIn "{dest}/share/doc/wake" "." readmeSource
     require Pass binfiles = binfilesResult
-    require Pass bininstall =
-        findFailFn releaseBin binfiles
+    require Pass bininstall = findFailFn releaseBin binfiles
     def outputs = readme, bininstall ++ datinstall
     Pass outputs
 
+
 # Replace @VERSION@ with 'release'
 def setVersion release file =
-    require Pass in =
-        source "{file}.in"
-    def script = "%
+    require Pass in = source "{file}.in"
+    def script =
+        "%
```

Comment:
The content of a multiline string should be indented wrt. the `"%` and `%"`.

Reply:
I'm not too familiar with `"%` strings, does adding indentation not change the real value of the string literal?

Reply:
Looking at the quickref, it seems like adding indentation here would definitely change the value of the multiline string. I don't think the formatter should be changing the value of the string even if its probably harmless

Reply:
It would not change the value of the string.

Reply:
These format the same and are the only two ways I think we should accept formatting multi-line strings:
```
export def hax1 = "%
  fun day.
  happy day.
%"
export def hax2 =
  "%
    fun day.
    happy day.
  %"
```

Reply:
SG, this should be an easy change. I was just being cautious around editing the stuff inside the quotes